src/librustc_data_structures/transitive_relation.rs

```diff
@@ -11,7 +11,7 @@ use std::mem;
 mod tests;
 
 #[derive(Clone, Debug)]
-pub struct TransitiveRelation<T: Clone + Debug + Eq + Hash> {
+pub struct TransitiveRelation<T: /*Clone + Debug +*/ Eq + Hash> {
```

Comment:
Why comment?

Reply:
Whoops, good catch. I have fixed and updated.